wordclock.py

```diff
@@ -60,11 +59,18 @@ def __init__(self):
         logging.info("Software version: " + self.currentGitHash)
 
         self.config = loadConfig(self.basePath)
+        
+        self.developer_mode_macos_active = self.config.getboolean('wordclock', 'developer_mode_macos')
+           
+        if self.developer_mode_macos_active:
+            # must be created first
+            import wx        
+            self.app = wx.App()
 
         # Create object to interact with the wordclock using the interface of your choice
         self.wci = wci.event_handler()
 
-        self.developer_mode_active = self.config.getboolean('wordclock', 'developer_mode')
+        self.developer_mode_active = self.config.getboolean('wordclock', 'developer_mode') or self.config.getboolean('wordclock', 'developer_mode_macos')
```

Comment:
I'd prefer to have one unique approach to enable developer mode. How about enabling wx.App() when "developer_mode" is set (removing developer_mode_macos).

Reply:
resolved